.travis.yml

```diff
@@ -37,6 +37,7 @@ matrix:
       services: xvfb
 
 install:
+  - "sudo apt-get install graphviz"
```

Comment:
Is this needed for testing? We're using Travis CI only for testing, and using readthedocs for building document.

Reply:
If needed, please use APT addon of Travis CI because `sudo` feature is disabled.
https://docs.travis-ci.com/user/installing-dependencies/#installing-packages-with-the-apt-addon

Reply:
It is needed, because Travis also runs the tox environment for generating the docs. This is why the first commit 
https://github.com/sphinx-doc/sphinx/pull/6164/commits/1f136215c18fc75eb9402ec7da083f9022703d71 failed the CI, see
https://travis-ci.org/sphinx-doc/sphinx/builds/504521299?utm_source=github_status&utm_medium=notification

Also, Travis now *always* provides sudo (`sudo: false` has no effect, see https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration). Hence, installing graphviz in the second commit does work, which is why the CI is now passing: https://travis-ci.org/sphinx-doc/sphinx/builds/504845281?utm_source=github_status&utm_medium=notification. In light of this, do you still want me to use the APT add-on instead?

You mention readthedocs. I don't see a `readthedocs.yml` file to configure the RTD environment. I don't know if the RTD server has graphviz installed, but if not, building the documentation will fail without some change to the configuration.

Reply:
Oh sorry, I don't know the change. Okay, let's keep as is. (we need to remove `sudo: false` instead).

About readthedocs, I'll take a look after merging this. And I'll change settings if needed.